modules/claims_api/app/services/claims_api/power_of_attorney_request_service/create_request.rb

```diff
@@ -7,6 +7,7 @@
 require 'bgs_service/vnp_ptcpnt_addrs_service'
 require 'bgs_service/vnp_ptcpnt_phone_service'
 require 'bgs_service/vnp_ptcpnt_service'
+require 'concurrent-ruby'
```

Comment:
Using concurrent-ruby for _some_ thread safety and readable syntax (and it is already in our Gemfile).

I considered Typhoeusʼ Hydra API, but it seems overly complex for our use case. I also considered Rubyʼs native Threads, but it does not have thread safety.